presto-main/src/main/java/com/facebook/presto/cost/HistoryBasedPlanStatisticsCalculator.java

```diff
@@ -215,6 +218,16 @@ private PlanNodeStatsEstimate getStatistics(PlanNode planNode, Session session,
         return delegateStats;
     }
 
+    @VisibleForTesting
+    public ConfidenceLevel getEnumConfidenceLevel(PlanStatistics predictedPlanStatistics)
```

Comment:
This can be a static function?

Reply:
I would move this to PlanStatistics.  

I would either make it non-static method on plan statistics like 
```
public ConfidenceLevel getConfidenceLevel()
```
or would make it static and as follows and take just a double
```
public static ConfidenceLevel toConfidenceLevel(double confidence)
```